.gitignore

```diff
@@ -9,3 +9,6 @@ yarn.lock
 # project
 .tmp
 lib
+
+# Mac OSX
+.DS_Store
```

Comment:
OS dependent stuff should be ignored in a global .gitignore file, but I'm fine with merging this in to avoid this in the future.